From bea0c6935989998fddec5e5c6d1d71da34ef8c70 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Wed, 18 Jan 2023 07:57:20 +0100
Subject: [PATCH] atlas-sw-probe: add more binaries

They were added in these commits [1] [2] and if they are not included,
the RIPE Atlas SW Probe does not work properly.

[1] https://github.com/RIPE-NCC/ripe-atlas-software-probe/commit/70ced29fc3217dd8d61e2b78506b6103ded100aa
[2] https://github.com/RIPE-NCC/ripe-atlas-software-probe/commit/71a4ff0e68c55464f766ddb9f1dfe21b22e530db

Fixes: https://github.com/openwrt/packages/issues/20338

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 net/atlas-sw-probe/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/atlas-sw-probe/Makefile b/net/atlas-sw-probe/Makefile
index bcef7a51f..b5ad4cf64 100644
--- a/net/atlas-sw-probe/Makefile
+++ b/net/atlas-sw-probe/Makefile
@@ -90,7 +90,7 @@ define Package/atlas-sw-probe/install
 	echo "prod" > $(1)/$(SCRIPTS_DIR)/state/mode
 
 	# Copy scripts
-	$(CP) $(PKG_BUILD_DIR)/bin/{ATLAS,common-pre.sh,common.sh,reginit.sh,resolvconf} $(1)/$(SCRIPTS_DIR)/bin/
+	$(CP) $(PKG_BUILD_DIR)/bin/{ATLAS,atlas_log.lib.sh,support.lib.sh,class.lib.sh,array.lib.sh,common-pre.sh,common.sh,json.lib.sh,reginit.sh,resolvconf} $(1)/$(SCRIPTS_DIR)/bin/
 	$(CP) $(PKG_BUILD_DIR)/bin/arch/{linux,openwrt-sw-probe} $(1)/$(SCRIPTS_DIR)/bin/arch/
 
 	# Create config info
-- 
2.34.1

